tests/src/python/test_qgsprocessinginplace.py

```diff
@@ -912,6 +913,26 @@ def test_unique_constraints(self):
 
         self.assertTrue(gpkg_layer.commitChanges())
 
+    def test_regenerate_fid(self):
+        """Test RegeneratePrimaryKey flag"""
+        temp_dir = QTemporaryDir()
```

Comment:
```suggestion

        temp_dir = QTemporaryDir()
```

Reply:
what am I missing? an extra line?

Reply:
yes, I though all our nice pre commit would have taken care of that but apparently not.